examples/nextjs-with-typescript/src/app/page.tsx

```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link';
```

Comment:
New landing page for the next example. Roughly analogous to [our next example app for elements](https://elements-demo-nextjs-mux.vercel.app/)